lib/spack/spack/mirrors/mirror.py

```diff
@@ -97,6 +97,11 @@ def display(self, max_len=0):
         binary = "b" if self.binary else " "
         print(f"{self.name: <{max_len}} [{source}{binary}] {url}")
 
+    def _process_spec_filters(self, key: str) -> List[str]:
```

Comment:
This should follow the naming convention we use for the other fields 

```suggestion
    def _get_spec_filters(self, key: str) -> List[str]:
```